content/blog/2024/anchor-position-polyfill-2.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Continued updates to the Anchor Position Polyfill
+sub: Performance and
+date: 2024-11-12
+image:
+  src: blog/2024/anchor2.jpg
+  alt: >
+    A pay-to-use pair of binoculars at a scenic overlook.
+author: james
+sponsors: true
+tags:
+  - Article
+  - Open Source
+  - Anchor Positioning
+  - CSS
+  - Polyfill
+summary: |
+  Our sponsors are supporting the continued development of the
+  CSS Anchor Positioning Polyfill.
+  Here's a summary of the latest updates.
+---
+{% import 'embed.macros.njk' as embed %}
+
+A lot has happened with Anchor Positioning since our [last update](/2024/07/02/anchor-position-polyfill/).
+
+With our [polyfill](https://github.com/oddbird/css-anchor-positioning), we've
+made a lot of progress in our goal of catching up to the spec, and based on the
+feedback we're hearing, people are finding it covers enough of their needs to be
+able to start using anchor positioning in production.
```

Comment:
"With our polyfill" sounds a bit awkward as a starting phrase. I'm not tied to this, but maybe something like:

```suggestion
We've made a lot of progress in our goal of catching up our
[polyfill](https://github.com/oddbird/css-anchor-positioning) to the spec, and
based on the feedback we're hearing, people are finding it covers enough of
their needs to be able to start using anchor positioning in production.
```